pkgs/tools/misc/chezmoi/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "chezmoi-${version}";
+  version = "1.3.0";
+
+  goPackagePath = "github.com/twpayne/chezmoi";
+
+  src = fetchFromGitHub {
+    owner = "twpayne";
+    repo = "chezmoi";
+    rev = "v${version}";
+    sha256 = "0dvdjx5khpw62lprn06k271xfc9fdrw4c1q74vd1vffaz60yfd8d";
+  };
+
```

Comment:
Can we add something like:

```
  buildFlagsArray = [
    "-ldflags=-s -w -X ${goPackagePath}/cmd.version=${version}"
  ];
```

Going by: https://github.com/twpayne/chezmoi/blob/master/.goreleaser.yaml#L11 this should give us a working `--version`